public/assets/reports/milestone1.tex

```diff
@@ -0,0 +1,258 @@
+  Milestone Report I}
+
+%for single author (just remove % characters)
+\author{
+  {\rm Sean Keever} \\
+  swkeever@uw.edu
+  \and
+  {\rm Yokesh Jayakumar} \\
+  mcmahjoh@uw.edu
+  \and
+  {\rm John McMahon} \\
+  karthj@uw.edu
+  % copy the following lines to add more authors
+  % \and
+  % {\rm Name}\\
+  %Name Institution
+} % end author
+
+\maketitle
+
+%-------------------------------------------------------------------------------
+\begin{abstract}
+  %-------------------------------------------------------------------------------
+  The goal of our project is to create a means of letting a user not only use an OS,
+  but to allow a user to see what is going on inside the OS and CPU.
+  To this end, our project aims to maximize availability to users by
+  providing a browser-based interface that lets the user visit a webpage and 
+  visualizing an OS from the moment he or she visits the page.
+\end{abstract}
+
+%-------------------------------------------------------------------------------
+\section{Introduction}
+%-------------------------------------------------------------------------------
+
+There are multiple implementations of operating systems made available via a web
+browser. Instead of re-implementing that functionality, we chose to find a project that
+has the low-level emulation from C to JavaScript already handled. This way, we can
+focus on solving a new problem: visualizing the OS and CPU.
+
+We looked at multiple different OS implementations and decided on risv-angel,
+which seemed to offer the
+functionality we needed without too much underlying complexity. This way, we could
+focus more attention to creating the visualization layers without having to worry
+too much about the emulation implementation details.
+
+%-------------------------------------------------------------------------------
+\section{Initial Design}
+%-------------------------------------------------------------------------------
+
+To start, we stripped the riscv-angel project to only show the Terminal.
+We got rid of all the elements and libraries that were not going to be used in our project.
+Before we started on our implementation, we prototyped the UI in Figma, shown in \ref{fig:proto1}.
+
+\begin{figure}[H]
+  \includegraphics[scale=0.3]{prototype1}
+  \caption{Initial prototype of design}
+  \label{fig:proto1}
+  \centering
+\end{figure}
+
+Figma allowed us to come up with a design before we implement anything in code.
+We aimed for a dashboard-like interface, showing the user all the most important
+information about the CPU and OS.
+Right now, we know we want to show:
+
+\begin{itemize}
+  \item Contents of registers (show values of registers in real time)
+  \item Contents of memory (let users zoom in on a particular region of memory)
+\end{itemize}
+
+\textbf{Question for the reader:} What else would be useful data to show to the user?
+We are having some trouble figuring out what exactly would be useful data to show.
```

Comment:
maybe add a sentence here saying that we are still determining what exactly we want to show in regards to memory?